src/DIRAC/StorageManagementSystem/DB/StorageManagementDB.py

```diff
@@ -1343,7 +1342,7 @@ def getCacheReplicasSummary(self, connection=False):
         resSummary = {}
         i = 1
         for status, se, numFiles, sumFiles in res["Value"]:
-            resSummary[i] = {"Status": status, "SE": se, "NumFiles": long(numFiles), "SumFiles": float(sumFiles)}
+            resSummary[i] = {"Status": status, "SE": se, int("NumFiles"): numFiles, "SumFiles": float(sumFiles)}
```

Comment:
I guess this was supposed to be:
```suggestion
            resSummary[i] = {"Status": status, "SE": se, "NumFiles": int(numFiles), "SumFiles": float(sumFiles)}
```